go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -380,21 +424,59 @@ func newSharedGatewayOpenFlowManager(nodeName, macAddress, gwBridge, gwIntf stri
 			gwIntf, stderr, err)
 	}
 
+	// Validate the numbers of the default OpenFlow rules and the port number of the associated ports, if they are
+	// we assume the OpenFlow rules are intact. Do nothing so ongoing network traffic won't be interrupted;
+	// Otherwise, flush existing OpenFlows and recreate default OpenFlows.
+	//
+	// Note that the numbers of default OpenFlows may be different depends on the IPv4/IPv6 mode or the other
+	// OVN k8s configurations
+
+	// One table 0 and three table 1 flows
```

Comment:
Comment is wrong, that does not equal 5 as specified on the line below.

Reply:
@alexanderConstantinescu Thanks for your review. I've fixed both of your comments, and repushed.